stats/index.js

```diff
@@ -12,9 +12,8 @@ for (var j = 0; j < 100; ++j) {
   let stages = []
 
   for (var i = 1; i < 4; ++i) {
-    let utxos = Simulation.generateTxos(20 / i, min, max)
-    let txos = Simulation.generateTxos(80 / i, min, max / 3)
-
+    let utxos = Simulation.generateTxos(20 / i, min, max, {106: 0.9, 300: 0.1})
```

Comment:
I am now thinking that all the simulation will be a little useless soon, since the segwit changes incentives anyway (so different strategy can end up a little better, because utxos are relatively less expensive)

but let's think about **that** later. Should I add the data from the table now?